test/Microsoft.AspNet.Mvc.Core.Test/ModelBinding/MutableObjectModelBinderTest.cs

```diff
@@ -417,10 +419,16 @@ public async Task
         {
             // Arrange
             var mockValueProvider = new Mock<IValueProvider>();
-            mockValueProvider.Setup(o => o.ContainsPrefixAsync(It.IsAny<string>()))
-                             .Returns(Task.FromResult(true));
+            mockValueProvider
+                .Setup(o => o.ContainsPrefixAsync(It.IsAny<string>()))
+                .Returns(Task.FromResult(true));
+
+            // Mock binder fails to bind all properties.
+            var mockBinder = new Mock<IModelBinder>();
+            mockBinder
+                .Setup(o => o.BindModelAsync(It.IsAny<ModelBindingContext>()))
+                .Returns((ModelBindingContext mbc) => Task.FromResult<ModelBindingResult>(null));
```

Comment:
BTW you can do this simpler by just using `.Returns(Task.FromResult<ModelBindingResult>(null));`